<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Page2</class>
 <widget class="QWizardPage" name="Page2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>360</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <widget class="QListWidget" name="fileList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>182</y>
     <width>436</width>
     <height>131</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::ContiguousSelection</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>319</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Input Data Type</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="addFileButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>157</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Browse File(s)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>158</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Basis data files:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeFileButton">
   <property name="geometry">
    <rect>
     <x>322</x>
     <y>157</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove File(s)</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="outputVDFtext">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>114</y>
     <width>436</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextEditorInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="vdfLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>88</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Output VDF File:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browseOutputVdfFile">
   <property name="geometry">
    <rect>
     <x>343</x>
     <y>87</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>13</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Create VDF</string>
   </property>
  </widget>
  <widget class="QLabel" name="selectFilePixmap">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>11</y>
     <width>59</width>
     <height>56</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="dataTypeComboBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>138</x>
     <y>323</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MOM</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>POP</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ROMS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>WRF</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CAM</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>GRIMs</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="processIndicator">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Analyzing Files...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
